crates/ruff_linter/src/rules/ruff/rules/sequence_sorting.rs

```diff
@@ -518,8 +522,11 @@ fn collect_string_sequence_lines(
             Tok::Comment(_) => {
                 line_state.visit_comment_token(subrange);
             }
-            Tok::String { value, .. } => {
-                line_state.visit_string_token(value, subrange);
+            Tok::String { .. } => {
+                let Some(string_value) = string_items_iter.next() else {
+                    unreachable!("Expected the number of string tokens to be equal to the number of string items in the sequence");
```

Comment:
nit: this is a very long line ;)

```suggestion
                    unreachable!(
                        "Expected the number of string tokens to be equal \
                        to the number of string items in the sequence"
                    );
```